src/editor/map/context_manager.cpp

```diff
@@ -52,10 +52,12 @@
 
 #include <memory>
 #include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
 
 namespace editor {
 
-static std::vector<std::string> saved_windows_;
+static std::vector<context_ptr> saved_contexts_;
+static int last_context_ = 0;
```

Comment:
Put these and the static functions in an anonymous namespace.

Reply:
Moved.

Reply:
👍 You don’t need to add the extra level of indent, though. Also you can keep the anon namespace inside the `editor` namespace.